src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -153,23 +153,12 @@ private static ArmorStand getArmorStand(@Nonnull Block projector, boolean create
             return null;
         }
 
-        ArmorStand hologram = spawnArmorStand(l);
+        ArmorStand hologram = ArmorStandUtils.spawnArmorStand(l);
+        hologram.setCustomNameVisible(true);
```

Comment:
Given you have now moved this to a util method, could we perhaps add overloads to that method to return a armorstand with a display name/visible name? Optional, however